packages/odie-client/src/components/message/message-content.tsx

```diff
@@ -27,50 +29,72 @@ export const MessageContent = forwardRef<
 			isLastFeedbackMessage,
 			isLastMessage,
 			isLastUserMessage,
+			isNextMessageFromSameSender,
+			displayChatWithSupportLabel,
 		},
 		ref: ForwardedRef< HTMLDivElement >
 	) => {
+		const { shouldUseHelpCenterExperience } = useOdieAssistantContext();
 		const isUser = message.role === 'user';
+		const isWapuu = message.role === 'bot';
+		const isHuman = message.role === 'business';
+
 		const messageClasses = clsx(
 			'odie-chatbox-message',
-			isUser ? 'odie-chatbox-message-user' : 'odie-chatbox-message-wapuu',
+			isUser && 'odie-chatbox-message-user',
+			isWapuu && 'odie-chatbox-message-wapuu',
```

Comment:
Wapuu is no longer a thing. We should be using something more generic like `odie` or `bot`. May seem trivial, but the name of all the pieces on this project has made it difficult to understand and explain (odie, wapuu, help center, zendesk, smooch, sunshine, etc)

Reply:
I've renamed to `bot`, it makes things simpler and more readable.